cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -140,26 +142,58 @@ func (s *server) addArchiveStorage(
 		return nil
 	}
 
-	f, err := jaegerstorage.GetStorageFactory(s.config.Storage.TracesArchive, host)
```

Comment:
Do we still need this method anywhere? If yes, can that code be upgraded as well? If no, I would remove it.